reconcile/github_org.py

```diff
@@ -33,29 +46,47 @@
 """
 
 
+def get_config():
+    gqlapi = gql.get_api()
+    orgs = gqlapi.query(ORGS_QUERY)['orgs']
+
+    config = {'github': {}}
+    for org in orgs:
+        org_name = org['name']
+        org_token = org['token']
+        token = vault_client.read(org_token['path'], org_token['field'])
+        org_config = {'token': token, 'managed_teams': org['managedTeams']}
+        config['github'][org_name] = org_config
+
+    return config
+
+
 def fetch_current_state(gh_api_store):
     state = AggregatedList()
 
     for org_name in gh_api_store.orgs():
         g = gh_api_store.github(org_name)
         raw_gh_api = gh_api_store.raw_github_api(org_name)
+        managed_teams = gh_api_store.managed_teams(org_name)
+        # if 'managedTeams' is not specified
+        # we manage all teams
+        is_managed = managed_teams is None
```

Comment:
maybe it makes sense to do:

`is_managed = managed_teams is None or managed teams == []`

Reply:
(in case someone defines an empty `managedTeams` )

Reply:
actually, better:
`is_managed = managed_teams is None or len(managed teams) == 0`

Reply:
thanks!